src/components/ha-attribute-value.ts

```diff
@@ -64,7 +64,7 @@ class HaAttributeValue extends LitElement {
     pre {
       font-family: inherit;
       font-size: inherit;
-      margin: 0px;
+      margin: 0;
```

Comment:
Just curious - are there any considerations for this? I usually use units with zero values, but I am not a css expert .

Reply:
No issues in this case. Lengths in CSS can be zero with no units. It's a habit I've had for like 15 years, I read somewhere that it was a "best practice" back then and it stuck. It's potentially a problem with using calc() and some other "newer" CSS features, but it's more of a personal preference than anything. I don't feel super strongly about it, especially on an open source project that I am just a contributor to. I'm happy to add the units back if that's the general preference for HA.

Reply:
I use zero without units too, so 🤷‍♂️